l1-contracts/test/fees/FeeRollup.t.sol

```diff
@@ -181,9 +212,8 @@ contract FeeRollupTest is FeeModelTestPoints, DecoderBase {
   }
 
   function test_BigBrainTime() public {
```

Comment:
may aswell continue with the naming policing while im here

Reply:
😭